runtime/vm/createramclass.cpp

```diff
@@ -1944,16 +1944,31 @@ loadFlattenableFieldValueClasses(J9VMThread *currentThread, J9ClassLoader *class
 	UDATA flattenableFieldCount = 0;
 	bool eligibleForFastSubstitutability = true;
 
+	/* Preload classes marked for preloading */
```

Comment:
Can you add a comment here saying that both the Q type check and preload classes are needed for things to work now because javac is not updated yet. In the future Q type check will be removed.